src/index.js

```diff
@@ -82,28 +81,32 @@ function issue(e) {
     attributes[input.dataset.attribute] = input.value;
   }
 
-  // Build the request.
-  var request = {
-    type: 'issuing',
-    credentials: [
+  // Issue, by showing a popup.
+  const request = {
+    '@context': 'https://irma.app/ld/request/issuance/v2',
+    'credentials': [
       {
-        credential: credentialID,
-        attributes: attributes,
+        'credential': credentialID,
+        'attributes': attributes,
       },
     ]
   };
-
-  // Issue, by showing a popup.
   console.log('issuing test attributes:', request);
-  irma.startSession(IRMA_SERVER, request, IRMA_AUTHMETHOD, IRMA_KEY, IRMA_REQUESTORNAME)
-    .then(function(pkg) {
-      return irma.handleSession(pkg.sessionPtr, {server: IRMA_SERVER, token: pkg.token, method: 'popup', language: LANG});
-    }).then(function(result) {
-      console.log('test attributes issued:', result);
-      return result;
-    }).catch(function(err) {
-      console.error('failed to issue test attributes:', err);
-    });
+
+  irma.newPopup({
+    debugging: true,
```

Comment:
Do you actually want verbose printing to the console log? Otherwise you can simply remove this option (or explicitly set it to `false` does the same; `false` is the default value).